README.md

```diff
@@ -2,3 +2,81 @@
 
 # Programs Branch Grant Programs (PBGP)
 
+<application description>
+## Prerequisites
+
+
+- Node.JS vX or newer
```

Comment:
I think we're using 16 and it doesn't work with 14! So link to Node 16 would be nice!